src/com/dre/brewery/Brew.java

```diff
@@ -332,12 +336,21 @@ public void distillSlot(ItemStack slotItem, PotionMeta potionMeta) {
 
 			addOrReplaceEffects(potionMeta, getEffects(), quality);
 			potionMeta.setDisplayName(P.p.color("&f" + recipe.getName(quality)));
-			slotItem.setDurability(PotionColor.valueOf(recipe.getColor()).getColorId(canDistill()));
+			if (P.use1_9) {
+				potionMeta.setMainEffect(PotionColor.valueOf(recipe.getColor()).getEffect());
+			} else {
+				slotItem.setDurability(PotionColor.valueOf(recipe.getColor()).getColorId(canDistill()));
+			}
+
 		} else {
 			quality = 0;
 			removeEffects(potionMeta);
 			potionMeta.setDisplayName(P.p.color("&f" + P.p.languageReader.get("Brew_DistillUndefined")));
-			slotItem.setDurability(PotionColor.GREY.getColorId(canDistill()));
+			if (P.use1_9) {
+				potionMeta.setMainEffect(PotionColor.GREY.getEffect());
+			} else {
+				slotItem.setDurability(PotionColor.GREY.getColorId(canDistill()));
+			}
```

Comment:
In 1.9 the canDistill() method is not used. It made sure that only potions that can be distilled are processed by the brewer. How is this handled in 1.9? Does the new Distill Tracker completely take care of that?